requirements-dev.txt

```diff
@@ -4,18 +4,18 @@
 #
 #    pip-compile --output-file=requirements-dev.txt requirements-dev.in
 #
-ansible-compat==3.0.1
+ansible-compat==3.0.2
     # via molecule
 ansible-core==2.14.4
     # via
     #   -r requirements-dev.in
     #   ansible-compat
     #   ansible-lint
-ansible-lint==6.14.4
+ansible-lint==6.14.6
     # via -r requirements-dev.in
 arrow==1.2.3
     # via jinja2-time
-attrs==22.2.0
+attrs==23.1.0
```

Comment:
Here's the [changelog](https://github.com/python-attrs/attrs/releases).

It contains one backwards